contribute/architecture/backend/services.md

```diff
@@ -107,31 +107,31 @@ func InitializeForTest(cla setting.CommandLineArgs, opts Options, apiOpts api.Se
 
 ## Background services
 
-A background service is a service that runs in the background of the lifecycle between Grafana starts up and shutdown. If you want a service to be run in the background your Service should satisfy the `registry.BackgroundService` interface and add it as argument to the [ProvideBackgroundServiceRegistry](/pkg/server/backgroundsvcs/background_services.go) function and add it as argument to `NewBackgroundServiceRegistry` to register it as a background service.
+A background service is a service that runs in the background of the lifecycle between Grafana startup and shutdown. If you want a service to be run in the background, your Service should satisfy the `registry.BackgroundService` interface and add it as argument to the [ProvideBackgroundServiceRegistry](/pkg/server/backgroundsvcs/background_services.go) function and add it as argument to `NewBackgroundServiceRegistry` to register it as a background service.
 
 You can see an example implementation above of the Run method.
 
 ## Disabled services
 
-If you want to guarantee that a background service is not run by Grafana when certain criteria is met/service is disabled your service should satisfy the `registry.CanBeDisabled` interface. When the service.IsDisabled method return false Grafana would not call the service.Run method.
+If you want to guarantee that a background service is not run by Grafana when certain criteria are met/service is disabled, your service should satisfy the `registry.CanBeDisabled` interface. When the service.IsDisabled method returns false, Grafana will not call the service.Run method.
```

Comment:
when certain criteria are met or the service is inactive? deactivated? It'd be good to choose a word other than 'disabled'.